pkg/kubelet/config/etcd.go

```diff
@@ -61,6 +61,10 @@ func (s *sourceEtcd) run() {
 	boundPods := api.BoundPods{}
 	err := s.helper.ExtractObj(s.key, &boundPods, false)
 	if err != nil {
+		if (tools.IsEtcdNotFound(err)) {
+			glog.V(4).Infof("etcd failed to retrieve the value for the key %q. Error: %v", s.key, err)
```

Comment:
May be overkill, but I would pull out into a msg:

```go
msg := fmt.Sprintf("etcd failed to retrieve the value for the key %q. Error: %v", s.key, err)
if tools.IsEtcdNotFound(err) {
  glog.V(4).Infof(msg)
  return
}
glog.Errorf(msg)
return
```

Then again, I'm unsure if minute refactoring like this is used or frowned upon for this codebase.